a3p-integration/proposals/z:acceptance/test-lib/psm-lib.js

```diff
@@ -436,18 +436,17 @@ const extractBalance = (balances, targetDenom) => {
  * @param {number} expectedBalance
  */
 export const tryISTBalances = async (t, actualBalance, expectedBalance) => {
-  const firstTry = await t.try(
-    (tt, actual, expected) => {
-      tt.deepEqual(actual, expected);
-    },
-    actualBalance,
-    expectedBalance,
-  );
+  const firstTry = await t.try(tt => {
+    tt.is(actualBalance, expectedBalance);
+  });
+  if (firstTry.passed) {
+    firstTry.commit();
+    return;
+  }
 
-  if (!firstTry.passed) {
-    firstTry.discard();
-    t.deepEqual(actualBalance + 200000, expectedBalance);
-  } else firstTry.commit();
+  firstTry.discard();
+  t.log('tryISTBalances assuming no batched IST fee', firstTry.errors);
+  t.is(actualBalance + 200_000, expectedBalance);
```

Comment:
consider defining a const for this magic number

Reply:
Done.